getting-started/installation-windows.rst

```diff
@@ -90,18 +97,31 @@ Install EPICS
 
 There should be lots of warnings, but no error. 
 
-Test EPICS
-----------
+Check EPICS Installation
+------------------------
 
 Run ``softIoc`` and, if everything is ok, you should see an EPICS prompt.
 
 ::
 
-    softIoc
-    epics>
+    $ /home/Hitesh/base-7.0.3.1/bin/windows-x64-mingw/softIoc
+    $ epics>
```

Comment:
1) Name is left there in hurry, everywhere else I avoided it. I will correct it with 'user'. 
2) For path like "c:\epics,,," : in Msys environment paths are like linux, so it starts with /home. 
3) For $HOME : while preparing documentation I tried to simplify for very beginner level users. putting $HOME might confuse some winows user. I will try to avoid name and simplify it.

All other points I would modify as per your suggestion

Reply:
Modification done and comitted.